packages/as-types/assembly/__tests__/serializer.spec.ts

```diff
@@ -20,10 +20,15 @@ import {
 } from '../serialization';
 
 describe('Serialization tests', () => {
-  it('ser/deser string', () => {
+  it('ser/deser Utf-16 string', () => {
     const str = 'Hello world 🙂';
     expect(bytesToString(stringToBytes(str))).toBe(str);
   });
+  it('ser/deser Utf-8 string', () => {
```

Comment:
It's rather an Ascii string ([a-zA-Z0-9] catalog to be precise). If you really want an UTF8 you should use some emoji stuff

Reply:
Not sure to get your point. there is a test with emoji just before this one.

So in the end, whats the conclusion? you think my proposal its not working? should we add some tests ? or you think this is confusing?

Reply:
I think my solution is working properly (unless someone prove me the opposite) and its saving some bytes in serialization.